packages/time-series-chart/src/Chart.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from "react";
+import { LineChart, ChartProps } from "./LineChart";
+import { options, series } from "./SampleData";
+import { ComponentController, DataSet, ColumnType } from "@dashbuilder-js/component-api";
+import { useState, useEffect, useCallback } from "react";
+import { Options, SingleSeries } from "./Data";
+
+// Default Values
+const DEFAULT_WIDTH = 400;
+const DEFAULT_HEIGHT = 200;
+const NOT_ENOUGH_COLUMNS_MSG =
+  "Time series component expects 2 columns: category(LABEL or TEXT or NUMBER or DATE) and series(NUMBER).";
+const FIRST_COLUMN_INVALID_MSG = "Wrong type for first column, it should be either LABEL, TEXT, NUMBER or DATE.";
+const SECOND_COLUMN_INVALID_MSG = "Wrong type for second column, it should be NUMBER.";
+
+enum Params {
+  WIDTH = "width",
```

Comment:
Could we make it fill the whole body (width 100%) by default?

Take as example Heatmap:

https://github.com/kiegroup/appformer/tree/master/dashbuilder/dashbuilder-shared/dashbuilder-js/packages/process-heatmap-component

It fill the whole page by default and the width/height setting can be controlled on Dashbuilder side, where we control the parent width and height